login/index.html

```diff
@@ -24,29 +25,40 @@
             height="132"
           />
         </a>
-        <form method="post">
+        <form method="get" action="/items">
           <div class="auth_form-el">
             <label for="login-id">이메일</label>
             <input
+              class="auth_input"
```

Comment:
```suggestion
              class="auth_input"
```

특정 선택자는 kebab-case, 특정 선택자는 snake-case로 naming convention을 혼용해서 사용하고 있습니다. 
기능에 지대한 영향을 끼치진 않지만 많은 시간을 쏟는 작명의 방식을 통일하는것도 일의 효율을 높일 수 있는데에 도움을 줍니다 :-)

Reply:
맞습니다. 코드 길이가 점점 많아지다보니 이름 짓는걸 통일하지 않았었는데 시간 들여서 통일해야할 것 같습니다.